ManatiCR/vitaminau
# VIT-74: Add unit catalog to the licensable technologies form.

# Task [VIT-74](https://manati.atlassian.net/browse/VIT-74)
Add unit catalog to the licensable technologies form.

# Installation
- `ahoy drush cr && ahoy drush cim -y`

# Test
- Go to the site at the following URL:
  - http://vitaminau.docker/form/webform-licensed-technologies
- Go to the "Generadores -> Unidad" field and verify that the unit field contains the unit category.

# CA
- Make the Generators Unit field point to Unit.
